corehq/apps/hqwebapp/views.py

```diff
@@ -393,6 +396,21 @@ def logout(req):
 def retrieve_download(req, domain, download_id, template="hqwebapp/file_download.html"):
     return soil_views.retrieve_download(req, download_id, template)
 
+@require_dropbox_session()
+def dropbox_upload(request, download_id):
```

Comment:
this may or may not belong in django soil. essentially this will allow us to sync any file to dropbox that uses the download base